Bottom: c76b355dcea7d1cf1ea82e02e95a95c385e45c11
Top:    00fe72466ddc003ac794adec8ce522b65275e7f7
Author: Jon Smirl <jonsmirl@gmail.com>
Date:   2012-07-01 09:01:57 -0400

From 6062de501a4e910188dc3e59d69179a58341ad95 Mon Sep 17 00:00:00 2001
Subject: ADC register defines


---

diff --git a/arch/arm/mach-lpc31xx/include/mach/registers.h b/arch/arm/mach-lpc31xx/include/mach/registers.h
index 45af7fc..01f4114 100644
--- a/arch/arm/mach-lpc31xx/include/mach/registers.h
+++ b/arch/arm/mach-lpc31xx/include/mach/registers.h
@@ -276,6 +276,14 @@
  * ADC_REG register definitions
  **********************************************************************/
 #define ADC_CON_REG            __REG (ADC_PHYS + 0x20)
+#define ADC_R0_REG             __REG (ADC_PHYS + 0x00)
+#define ADC_R1_REG             __REG (ADC_PHYS + 0x04)
+#define ADC_R2_REG             __REG (ADC_PHYS + 0x08)
+#define ADC_R3_REG             __REG (ADC_PHYS + 0x0C)
+#define ADC_CSEL_REG           __REG (ADC_PHYS + 0x24)
+#define ADC_INT_ENABLE_REG     __REG (ADC_PHYS + 0x28)
+#define ADC_INT_STATUS_REG     __REG (ADC_PHYS + 0x2C)
+#define ADC_INT_CLEAR_REG      __REG (ADC_PHYS + 0x30)
 
 /***********************************************************************
  * SYS_REG register definitions
